src/core/viz/parser.js

```diff
@@ -179,3 +179,78 @@ function cleanJsep() {
     jsep.addLiteral('true');
     jsep.addLiteral('false');
 }
+
+/**
+ * Remove comments from string
+ * - // line comments
+ * - /* block comments
+ * - ' " Keep comments inside single and double quotes
+ * Based on: https://j11y.io/javascript/removing-comments-in-javascript/
+ */
+export function cleanComments(str) {
+    var mode = {
+        singleQuote: false,
+        doubleQuote: false,
+        blockComment: false,
+        lineComment: false
+    };
+
+    // Adding chars to avoid index checking
+    str = ('_' + str + '_').split('');
+
+    for (var i = 0, l = str.length; i < l; i++) {
+
+        if (mode.singleQuote) {
+            if (str[i] === '\'' && str[i-1] !== '\\') {
+                mode.singleQuote = false;
+            }
+            continue;
+        }
+
+        if (mode.doubleQuote) {
+            if (str[i] === '"' && str[i-1] !== '\\') {
+                mode.doubleQuote = false;
+            }
+            continue;
+        }
+
+        if (mode.blockComment) {
+            if (str[i] === '*' && str[i+1] === '/') {
+                str[i+1] = '';
+                mode.blockComment = false;
+            }
+            str[i] = '';
+            continue;
+        }
+
+        if (mode.lineComment) {
+            if (str[i+1] === '\n' || str[i+1] === '\r') {
+                mode.lineComment = false;
+            }
+            if (i+1 < l) {
+                str[i] = '';
```

Comment:
Oh, maybe that's the reason for the split?

Reply:
Yep ^